### PR TITLE
Pass Annotations to PVC

### PR DESCRIPTION
**Pull request by dvaldivia**
This PR pushes down the Annotations for the Tenant to the underlying PVCs via their Claim Template

### CODE REVIEW

File: restapi/admin_tenants.go [hunk 1]
```diff
@@ -815,9 +815,18 @@ func getTenantCreatedResponse(session *models.Principal, params admin_api.Create
...
 		minInst.Spec.ServiceName = tenantReq.ServiceName
 	}
+	// add annotations
+	var annotations map[string]string
+	if len(tenantReq.Annotations) > 0 {
+		if minInst.Spec.Metadata == nil {
+			minInst.Spec.Metadata = &metav1.ObjectMeta{}
+		}
+		annotations = tenantReq.Annotations
+		minInst.Spec.Metadata.Annotations = annotations
```
Comment (kerneltime) [inline restapi/admin_tenants.go:825]: Applying the annotation to both PVC and the Tenant?

Comment (dvaldivia) [reply restapi/admin_tenants.go:825]: Yes, the requests Annotations for the Tenant are pushed down to the PVC

Comment (kerneltime) [reply restapi/admin_tenants.go:825]: We can split them later I guess..

File: restapi/admin_tenants.go [hunk 2]
```diff
@@ -1241,16 +1243,22 @@ func parseTenantZoneRequest(zoneParams *models.Zone) (*operator.Zone, error) {
...
+			Name: "data",
 		},
+		Spec: volTemp,
+	}
+	if len(annotations) > 0 {
+		vct.ObjectMeta.Annotations = annotations
+	}
+
+	zone := &operator.Zone{
+		Name:                zoneParams.Name,
```
Comment (Alevsk) [inline restapi/admin_tenants.go:1258]: Can `zoneParams` be null at some point? I observed its being received as a pointer

Comment (dvaldivia) [reply restapi/admin_tenants.go:1258]: it's validated before calling this function so it can't be null

**Pull request merged**